rbkit-lib/cpu/cpunode.h

```diff
@@ -0,0 +1,49 @@
+#ifndef NODE_H
+#define NODE_H
+#include <QString>
+#include <vector>
+#include <QDebug>
+
+class Node
+{
+    QString methodName,
+            label,
+            filename,
+            thread_id;
+
+    int singleton_method;
+    double line_no;
+
+    QList<Node*> calledBy;
+    QList<Node*> calls;
+public:
+    Node(QString methodName,
+         QString label,
+         QString filename,
+         QString thread_id,
+         int line_no,
+         int singleton_method);
+
+    void updateCalls(Node*);
```

Comment:
use references as much as possible. avoid using pointers. this can be written as `void updateCalls(Node& node);`